slsim/lens_pop.py

```diff
@@ -1,378 +1,70 @@
-        :type slhammocks_config: string
-        :param sky_area: Sky area over which lens population will be simulated. If
-         sky_area is not None, number of source sample and deflector sample within a
-         source_sky_area and deflector_sky_area will be scaled to the sky_area.
-         This will allow us to simulate lens population over a large sky area without
-         further significant computational cost.
-        :param source_sky_area: Sky area over which sources are sampled. Must be in
-         units of solid angle. If None, source_sky_area will be equal to sky_area.
-        :type source_sky_area: `~astropy.units.Quantity`
-        :param deflector_sky_area: Sky area over which deflectors are sampled. Must be
-         in units of solid angle. If None, deflcetor_sky_area will be equal to sky_area.
-        :type deflector_sky_area: `~astropy.units.Quantity`
-        :type sky_area: `~astropy.units.Quantity`
-        :param filters: filters for SED integration
-        :type filters: list of strings or None
-        :param cosmo: cosmology object
-        :type cosmo: `~astropy.cosmology.FLRW`
-        :param source_light_profile: keyword for number of sersic profile to use in
-         source light model. It is necessary to recognize quantities given in the source
-         catalog.
-        :type source_light_profile: str . Either "single" or "double" .
-        :param catalog_type: type of the catalog. If someone wants to use scotch
-         catalog, they need to specify it.
-        :type catalog_type: str. eg: "scotch"
-        :param catalog_path: path to the source catalog. If None, existing source
-         catalog within the slsim will be used. We have used small subset of scotch
-         catalog. So, if one wants to use full scotch catalog, they can set path to
-         their path to local drive.
-        :param lightcurve_time: observation time array for lightcurve in unit of days.
-        :type lightcurve_time: array
-        :param sn_type: Supernova type (Ia, Ib, Ic, IIP, etc.)
-        :type sn_type: str
-        :param sn_absolute_mag_band: Band used to normalize to absolute magnitude
-        :type sn_absolute_mag_band: str or `~sncosmo.Bandpass`
-        :param sn_absolute_zpsys: Optional, AB or Vega (AB default)
-        :type sn_absolute_zpsys: str
-        :param los_config: configuration for line of sight distribution
-        :type los_config: LOSConfig instance
-        :param sn_modeldir: sn_modeldir is the path to the directory containing files
-         needed to initialize the sncosmo.model class. For example,
-         sn_modeldir = 'C:/Users/username/Documents/SALT3.NIR_WAVEEXT'. These data can
-         be downloaded from https://github.com/LSST-strong-lensing/data_public .
-         For more detail, please look at the documentation of RandomizedSupernovae
-         class.
-        :type sn_modeldir: str
+        Args:
```

Comment:
can you follow the :param input_name: and :type input_name: conventions here?

Reply:
Definitely can. Just curious, is there a specific reason we use the harder-to-read rEST format vs. the easier-to-read Google format, considering they are both supported by sphinx? One could argue that better docstring readability in the source code could promote contribution. :smile:

Reply:
no, this was mainly the initial setting and the current sphinx configuration. If you want to change the sphinx configuration, that would work, just would eventually need to update many docstrings, that's all

Reply:
If you're okay with that , then I'd love to push for it! Can add it as an issue for the refactor. :star_struck:

Reply:
I am ok with it :)